buildconfig/config_unix.py

```diff
@@ -137,7 +137,7 @@ def main():
     origincdirs = ['/include', '/include/SDL2']
     origlibdirs = ['/lib', '/lib64', '/X11R6/lib',
                    '/lib/i386-linux-gnu', '/lib/x86_64-linux-gnu',
-                   '/lib/arm-linux-gnueabihf/', '/lib/aarch64-linux-gnu/']
+                   '/lib/arm-linux-gnueabihf/', '/lib/aarch64-linux-gnu/', '/lib/powerpc64le-linux-gnu']
```

Comment:
Minor nitpick: there should be a trailing `/` for the directory path string (this might not be critical to get this working, but there's no harm in having one incase some other part of the codebase does some naive filepath handling)
Also this can come on a newline for better formatting

Reply:
Thanks for the review.

Changed as requested.